src/SFML/Network/Packet.cpp

```diff
@@ -200,7 +200,7 @@ Packet& Packet::operator>>(std::uint32_t& data)
     if (checkSize(sizeof(data)))
     {
         std::memcpy(&data, &m_data[m_readPos], sizeof(data));
-        data = ntohl(data);
+        data = sf::swapEndian(data);
```

Comment:
```suggestion
        data = swapEndian(data);
```
You can drop the `sf::` namespace whenever you're calling this.